web/concrete/controllers/install.php

```diff
@@ -265,6 +268,24 @@ public function passedRequiredItems()
         }
     }
 
+    public function testCurlCaBundle()
+    {
+        if (function_exists('curl_init')) {
+            $ch = curl_init('https://www.concrete5.org');
+            curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+            curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, true);
+            curl_setopt($ch, CURLOPT_NOBODY, true);
+            $res = !(curl_exec($ch) === false &&
+                ((curl_errno($ch) === 60 && curl_error($ch) === 'SSL certificate problem: unable to get local issuer certificate')
```

Comment:
We shouldn't be doing a check against the human readable curl error, that's what the error number is for.